.ci/create-release-github.sh

```diff
@@ -1,7 +1,6 @@
 #!/bin/bash
 
 OPERATOR_VERSION=$(git describe --tags)
-echo "${GH_WRITE_TOKEN}" | gh auth login --with-token
 
 gh config set prompt disabled
 gh release create \
```

Comment:
Doesn't have to be this PR but I would remove this script since it's effectively one line (can and probably should use the value of the tag in the CI event rather than needing `git describe` for it)